Automattic/vip-go-mu-plugins
# Remove Bulk Edit Limitations

We no longer should be restricting these actions since they impact customer workflows.

## Changelog Description

### Removed Bulk Editing Restrictions

We've removed restrictions around bulk editing

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure you have lots of posts (e.g. 500)
1. Go to `wp-admin` > `Posts` > `Edit`
1. Increase the post count to 100
1. Verify that the bulk edit banner doesn't show up and you can bulk edit all posts
